doc improvement: Zigbee QSG (#6965)
nrfconnect/sdk-nrf: doc/nrf/ug_zigbee_qsg.rst

Requirements
************

> For this quick start guide, you need **three** development kits of the following types:

Reviewer: Nit: although there is no prohibition for using bold, we usually reserve bold for some very specific use cases (https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Typographical+conventions ), so you might want to consider using italics here (with single asterisks).

   On the development kit programmed with the light bulb sample, **LED 3** turns on when the light bulb joins the network.
   On the development kit programmed with the light switch sample, **LED 3** turns on when the device joins the network and **LED 4** turns on when the light switch finds a light bulb to control.

Reopening the network

>    If you find that you are having issues with your devices joining the Zigbee network, it may be because the connection needs to be reopened manually.

Reviewer 2: I think it would be more consistent to write `.. it may be because the network needs to be reopened...` instead if `..the connection..`

Author: Okay, I will make this change.

   On the development kit programmed with the light bulb sample, **LED 3** turns on when the light bulb joins the network.
   On the development kit programmed with the light switch sample, **LED 3** turns on when the device joins the network and **LED 4** turns on when the light switch finds a light bulb to control.

> .. note::

Reviewer: I would add this as an indented header, that is something like this:
```suggestion
Reopening the network

   If you find that...
```
And maybe add "(see below)" at the end of line 193 ("Zigbee network needs to be reopened").

Not that I'm not a fan of notes, but in this case, I find it a bit long and maybe a pseudo-subsection would be better.

> 

Reviewer: Remove this line to trigger the RST formatting for definition headers.

   If you find that you are having issues with your devices joining the Zigbee Network, it may be because the connection needs to be reopened manually.
   If the network is open, then **LED 3** on the developement kit programmed with the network coordinator sample will be on.
   If **LED 3** is not on, then the network is closed and needs to be reopened again.

>    To do this, press **Button 1** on the developement kit that has been programmed as the network coordinator.

Reviewer: ```suggestion
   To reopen the network, press **Button 1** on the developement kit that has been programmed as the network coordinator.
```
Just a nit, to avoid the deictic reference that might be a bit (just a bit) confusing.
